system/system_error_monitor/config/diagnostic_aggregator/localization.param.yaml

```
@@ -18,11 +18,11 @@
           type: diagnostic_aggregator/AnalyzerGroup
           path: performance_monitoring
           analyzers:
-            matching_score:
-              type: diagnostic_aggregator/GenericAnalyzer
-              path: matching_score
-              contains: ["ndt_scan_matcher"]
-              timeout: 1.0
+            # matching_score:
+            #   type: diagnostic_aggregator/GenericAnalyzer
+            #   path: matching_score
+            #   contains: ["ndt_scan_matcher"]
```

Instead of removing it, I think you should consider renaming it.
cc @tkimura4 

```suggestion
            #   contains: ["pose_estimator_score"]
```

---

If it's really impossible to have a common diagnostic item for multiple methods of localization, please add the reason to the PR description.